### PR TITLE
feat: send binary message to kernel

### PR DESCRIPTION
**Pull request by pbosio**
add code to be able to send binary messages from renderer to kernel

### CODE REVIEW

**Review by kuruk-mm — APPROVED** [review]
Looks great!

File: unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/WebSocketCommunication/DCLWebSocketService.cs [hunk 1]
```diff
@@ -32,6 +36,23 @@ private void SendMessageToWeb(string type, string message)
 #endif
     }
 
+    private void SendBinaryMessageToWeb(string sceneId, byte[] data)
+    {
+#if (UNITY_EDITOR || UNITY_STANDALONE)
```
Comment (AjimenezDCL) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/WebSocketCommunication/DCLWebSocketService.cs:41]: Why is this only relevant to editor/standalone?

**Review by AjimenezDCL — APPROVED** [review]
Looks great, just left a small question

File: unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/WebSocketCommunication/DCLWebSocketService.cs (hunk 1 above)
Comment (pbosio) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/WebSocketCommunication/DCLWebSocketService.cs:41]: that is for sending messages through websocket, which we are only using for editor and desktop version.
and the webgl message goes through `JSFunctions.jslib`

File: unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/WebSocketCommunication/DCLWebSocketService.cs [hunk 2]
```diff
@@ -32,6 +36,23 @@ private void SendMessageToWeb(string type, string message)
 #endif
     }
 
+    private void SendBinaryMessageToWeb(string sceneId, byte[] data)
```
Comment (kuruk-mm) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/WebSocketCommunication/DCLWebSocketService.cs:39]: Can we change this to `SendBinaryMessageToKernel`?

I know that we use `Web` like `WebInterface` and more in a lot of places. But I think `Kernel` is a better naming. Do you agree?
